### PR TITLE
docs(cx-navigation): Update Navigation Docs

### PR DESCRIPTION
**Pull request by misterjones**
* Perform standard doc cleanup.
* Change instances of CX/CanvasX to Vital.
* Add "Content Editor Details" where applicable.
* Add screenshots.
* Update doc structure from "CX Navigation" to "Vital Navigation."
* Make "Sub-Menu" spelling to be consistent with UI, and throughout doc.

### CODE REVIEW

**Review by gzeni-cit — APPROVED** [review]

File: packages/cx-navigation/doc/BurgerMenu.md [hunk 1]
```diff
@@ -9,28 +10,45 @@ The CX BurgerMenu Component provides a triggerable navigation menu for Mobile an
...
+layout.
 
-The Main Menu content is the same as the Top Navigation Menu on Desktop. We also apply the same WhereToBuy, UtilityMenu, and LanguageButton, compoments from Desktop Top Nav, but following a different order for the BurgerMenu.
+The Main Menu content is the same as the Top Navigation Menu on Desktop. We also apply the same
+`WhereToBuy`, `UtilityMenu`, and `LanguageButton` components from Desktop Top Nav, but following a
+different order for the Burger Menu.
 
-The Overlay is an empty div, defined as a sibling for the BurgerMenu wrapper, and it is necessary to provide a background layer for the BurgerMenu as it is opened. That layer is blur and requires some tailwind customizations with filter backdrop to achieve the desired design. By default, it is not possible to see the Overlay on Mobile since the BurgerMenu is full width, but it is visible on Tablet once it takes place only in part of the screen.
+The Overlay is an empty `<div>`, defined as a sibling for the `BurgerMenu` wrapper, and it is
+necessary to provide a background layer for the `BurgerMenu` as it is opened. That layer is blurred
```
Comment (hvanyo) [inline packages/cx-navigation/doc/BurgerMenu.md]: I would reword this as 
On tablet the burger menu only covers partial portion of the site and site itself is blurred to prevent user interaction with it.  The blur effect is done using Backdrop blur https://tailwindcss.com/docs/backdrop-blur and cutomized to your design requirements.
On mobile, the burger menu uses the full width of the viewport.

Comment (hvanyo) [inline packages/cx-navigation/doc/BurgerMenu.md]: I would reword this as 
On tablet the burger menu only covers partial portion of the site and site itself is blurred to prevent user interaction with it.  The blur effect is done using Backdrop blur https://tailwindcss.com/docs/backdrop-blur and cutomized to your design requirements.
On mobile, the burger menu uses the full width of the viewport.

File: packages/cx-navigation/doc/BurgerMenu.md [hunk 2]
```diff
@@ -9,28 +10,45 @@ The CX BurgerMenu Component provides a triggerable navigation menu for Mobile an
...
+and requires some Tailwind customizations with filter backdrop to achieve the desired design. By
+default, it is not possible to see the Overlay on Mobile since the `BurgerMenu` is full-width, but
+it is visible on Tablet as it takes place only on part of the screen.
 
 ## Content Editor Details
 
-There is no way to edit BurgerMenu components (we are adding `asStatic` to the whole component to make sure it is not editable). The components are editable on Desktop though.
+There is no way to edit `BurgerMenu` components (we are adding `asStatic` to the whole component to
+make sure it is not editable). The components are editable on Desktop though.
```
Comment (hvanyo) [inline packages/cx-navigation/doc/BurgerMenu.md]:
```suggestion
make sure it is not editable). The components are editable on Desktop  menu though.
```
```suggestion
make sure it is not editable). The components are editable on Desktop though.
```

File: packages/cx-navigation/doc/Menu.md [hunk 3]
```diff
@@ -22,27 +25,97 @@ In the end, the menu is rendered in the following structure:
 </Nav>
 ```
 
-The Wrapper is just an `<ul>` element. By using `asBodilessMenu()`, the default MenuClean provides an editable Item list, rendered as `<li>`. The Title inside is just an editable link in the clean component but the base/default tokens provided will replace it with CX MenuTitle.
+The Wrapper is simply an `<ul>` element. By using `asBodilessMenu()`, the default `MenuClean`
```
Comment (hvanyo) [inline packages/cx-navigation/doc/Menu.md]: simply?

Comment (hvanyo) [inline packages/cx-navigation/doc/Menu.md]: simply?

File: packages/cx-navigation/doc/Menu.md [hunk 4]
```diff
@@ -22,27 +25,97 @@ In the end, the menu is rendered in the following structure:
...
+Where made available by a Site Builder, you can add Menu Items to the Menu. Once added, you can:
+
+- [Add a Menu Title](./MenuTitle#addedit-menu-title);
+- [Add a Menu Link](#add-a-menu-link); and
+- [Add a Sub-Menu](./Sub-Menu#add-a-sub-menu).
+
+![Main Menu Item context menu](../assets/MainMenuItemContextMenu.jpg ':size=50%')
+
+?> **Note:** No (parent) Menu Item with children can be a Link; i.e., if you add a Sub-Menu to a
+Menu Item, the Menu Item cannot then be a Link.
```
Comment (hvanyo) [inline packages/cx-navigation/doc/Menu.md]: This is primarly a design choice and not technical choice.   If you want a menu item with subchildren to be a link, we recommend you a submenu indicators so that menu is accessible.    @Guilherme-Almeida-Zeni  is menu indicators in bodiless yes?

Comment (hvanyo) [inline packages/cx-navigation/doc/Menu.md]: This is primarly a design choice and not technical choice.   If you want a menu item with subchildren to be a link, we recommend you a submenu indicators so that menu is accessible.    @Guilherme-Almeida-Zeni  is menu indicators in bodiless yes?

File: packages/cx-navigation/doc/BurgerMenu.md (hunk 1 above)
Comment (misterjones) [reply packages/cx-navigation/doc/BurgerMenu.md]: ✅ Updated.

File: packages/cx-navigation/doc/BurgerMenu.md (hunk 2 above)
Comment (misterjones) [reply packages/cx-navigation/doc/BurgerMenu.md]: ✅ Updated.

File: packages/cx-navigation/doc/Menu.md (hunk 3 above)
Comment (misterjones) [reply packages/cx-navigation/doc/Menu.md]: Whoops. 😅 
I remember trying this, and not liking it; didn't realize I had left it in.

✅ Updated.

File: packages/cx-navigation/doc/Menu.md (hunk 4 above)
Comment (misterjones) [reply packages/cx-navigation/doc/Menu.md]: @hvanyo, ah, gotcha.
✅ I removed this note, as well as the adjoining one on the Sub-Menu page.

If indicators are in Bodiless, I can work with @Guilherme-Almeida-Zeni on documenting your recommendation.

**Comment by misterjones** [comment]
Moving PR to main repo: [PR #1569](https://github.com/johnsonandjohnson/Bodiless-JS/pull/1569)

Closing.
